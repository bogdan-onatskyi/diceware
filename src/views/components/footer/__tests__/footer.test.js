import React from 'react';

import Footer from '../footer';

describe("Footer component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <Footer/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it("should have 1 prop", () => {
        const propsStr = 'className';
        const props = [];
        for (let prop in Footer.propTypes) props.push(prop);
        expect(props.join(' ') === propsStr).toBe(true);
    });
});