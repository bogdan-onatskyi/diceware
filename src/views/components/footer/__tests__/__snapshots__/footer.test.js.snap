// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match saved snapshot 1`] = `
<footer
  className="footer"
>
  <span
    className="footer--icon"
  />
  +38 (097) 499-73-82 e‑mail: 
  <a
    href="mailto: gentoo.user@ukr.net"
  >
    Богдан Онацкий
  </a>
</footer>
`;
